uber/templates/jobs/staffers.html

```diff
@@ -76,4 +98,48 @@
     [L]: <i>Staffers marked with a "L" are working load-in and/or load-out</i>
 </div>
 
+
+{% if c.HAS_ACCOUNTS_ACCESS %}
+<script src="../static/js/jquery.form.js"></script>
+<script>
+var list_of_ids = [];
+
+$("#admin_settings").ajaxForm(function(message){
+    toastr.info(message);
+    setTimeout(function(){
+        location.reload(true);
+    }, 1500);
+});
+
+var checkAll = function(){
+    $.each( $("input[name^='id']"), function () {
+      $(this).prop("checked", true);
+    });
+};
+
+var uncheckAll = function(){
+    $.each( $("input[name^='id']"), function () {
+      $(this).prop("checked", false);
+    });
+};
+
+var adminList = function(id){
```

Comment:
You're not handling the case where someone checks a checkbox and then decides the person doesn't need an admin account and unchecks it.

You can either add/remove it from the list based on the current `checked` state of the element, or you can stop maintaining this list and grab all the checked values on form submit. (I'd recommend the latter approach.)